js/app/controllers/content-product.ng

```diff
@@ -77,6 +77,124 @@ Set-GceInstance "instance-name" -RemoveMetadata "newKey"
 Set-GceInstance "instance-name" -RemoveTag "beta" -AddTag "alpha"</pre>  
   </div>
 
+    <!-- Container Engine -->
+  <div ng-switch-when="google-cloud-container">
+    <p>
+      Google Container Engine is a powerful cluster manager and orchestration
+      system for running your Docker containers. Container Engine schedules
+      your containers into the cluster and manages them automatically based
+      on the requirements you define (such as CPU and memory). It's built
+      on the open source Kubernetes system, giving you the flexibility to take
+      advantage of on-premises, hybrid, or public cloud infrastructure.
+    </p>
+
+    <h2>Container Clusters</h2>
+    <p>
+      You can create a cluster by first creating a <code>NodeConfig</code>
+      object with <cmdlet-ref name="New-GkeNodeConfig"></cmdlet-ref> cmdlet.
+      After that, you can pass in the <code>NodeConfig</code> object to the
+      <cmdlet-ref name="Add-GkeCluster"></cmdlet-ref> cmdlet. It will then create
+      a cluster whose node pools will have their configurations set from the
+      <code>NodeConfig</code> object.
+    </p>
+    <pre># Creates a Container Engine Node Config with image type CONTAINER_VM
```

Comment:
Does an initial line end at the very start of the `<pre>` block look bad? If not, adding one might make it slightly easier to read.